File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-1.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        #make a list for the unique numbers
        #to do this i make a set this takes all elements that are the same out and gives me a set 
        temp = set(nums)
        #i convert back to a list
        unique = list(temp)
        #sort
        unique.sort()
        #get the size of the unique elements 
        length = len(unique)

        #for the elements inbetween 0 and len set to unqiue // change the original  list
        nums[ :length] = unique
        return length

File: Data Structures & Algorithms/find-duplicate-integer/submission-1.cpp
class Solution {
public:
    int findDuplicate(vector<int>& nums) {
        for(size_t i{};i<nums.size();i++){
            for(size_t j = i + 1;j<nums.size();j++){
                if(nums[i] == nums[j])
                    return nums[i];
            }
        }
    }
};
